README.md

```diff
@@ -27,7 +27,8 @@
 [![Twitter Follow](https://img.shields.io/twitter/follow/ApacheAirflow.svg?style=social&label=Follow)](https://twitter.com/ApacheAirflow)
 [![Slack Status](https://img.shields.io/badge/slack-join_chat-white.svg?logo=slack&style=social)](https://apache-airflow-slack.herokuapp.com/)
 
-Apache Airflow (or simply Airflow) is a platform to programmatically author, schedule, and monitor workflows.
+[Apache Airflow](https://airflow.apache.org/docs/stable/) (or simply Airflow) is a platform to programmatically author, schedule, and monitor
```

Comment:
Comment. Maybe - since we do not have it in "airflow.apache.org" - we should also add links to "readthedos" with latest documentation ? I think we have a lot of new documentation in master that is not used by anyone and people do not even know it exists.

Reply:
How should we do this to avoid confusion?

Reply:
We have links to docs in "Getting Started" section".  https://github.com/apache/airflow/blob/a2fa056ce9fcb49a5e443a86020226a5e7e5e6df/README.md#getting-started

Reply:
Did not realise that :)